<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>numericPad</class>
 <widget class="QWidget" name="numericPad">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>446</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image:url(:/transparent.png);</string>
  </property>
  <widget class="QFrame" name="calcForm">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>500</width>
     <height>446</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-image:url(/transparent.png);
background-color: qlineargradient(spread:pad, x1:0.75, y1:0.824, x2:0.037, y2:0.05, stop:0 rgba(30,30,30, 200), stop:1 rgba(200, 200, 200, 120));
border: 1px solid  rgb(74, 74, 74);
border-radius:10px;
border-right-color: rgba(255, 255, 153,0);
border-left-color: rgba(255, 255, 153,0);
border-top-color: rgba(255, 255, 153,0);
</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="push1">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>120</y>
      <width>85</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bitstream Charter</family>
      <pointsize>22</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-image:url(/transparent.png);
background-color: rgba(139, 139, 139, 50);
border: 2px solid  rgb(0, 0, 0);
border-radius:4px;
border-right-color: rgba(0, 0, 0,40);
border-left-color: rgba(255, 255, 255,100);
border-top-color: rgba(255, 255, 153,0);
</string>
    </property>
    <property name="text">
     <string notr="true">1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="push2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>120</y>
      <width>85</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bitstream Charter</family>
      <pointsize>22</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-image:url(/transparent.png);
background-color: rgba(139, 139, 139, 50);
border: 2px solid  rgb(0, 0, 0);
border-radius:4px;
border-right-color: rgba(0, 0, 0,40);
border-left-color: rgba(255, 255, 255,100);
border-top-color: rgba(255, 255, 153,0);
</string>
    </property>
    <property name="text">
     <string notr="true">2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="push3">
    <property name="geometry">
     <rect>
      <x>205</x>
      <y>120</y>
      <width>85</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bitstream Charter</family>
      <pointsize>22</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-image:url(/transparent.png);
background-color: rgba(139, 139, 139, 50);
border: 2px solid  rgb(0, 0, 0);
border-radius:4px;
border-right-color: rgba(0, 0, 0,40);
border-left-color: rgba(255, 255, 255,100);
border-top-color: rgba(255, 255, 153,0);
</string>
    </property>
    <property name="text">
     <string notr="true">3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="push6">
    <property name="geometry">
     <rect>
      <x>205</x>
      <y>180</y>
      <width>85</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bitstream Charter</family>
      <pointsize>22</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-image:url(/transparent.png);
background-color: rgba(139, 139, 139, 50);
border: 2px solid  rgb(0, 0, 0);
border-radius:4px;
border-right-color: rgba(0, 0, 0,40);
border-left-color: rgba(255, 255, 255,100);
border-top-color: rgba(255, 255, 153,0);
</string>
    </property>
    <property name="text">
     <string notr="true">6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="push4">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>180</y>
      <width>85</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bitstream Charter</family>
      <pointsize>22</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-image:url(/transparent.png);
background-color: rgba(139, 139, 139, 50);
border: 2px solid  rgb(0, 0, 0);
border-radius:4px;
border-right-color: rgba(0, 0, 0,40);
border-left-color: rgba(255, 255, 255,100);
border-top-color: rgba(255, 255, 153,0);
</string>
    </property>
    <property name="text">
     <string notr="true">4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="push5">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>180</y>
      <width>85</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bitstream Charter</family>
      <pointsize>22</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-image:url(/transparent.png);
background-color: rgba(139, 139, 139, 50);
border: 2px solid  rgb(0, 0, 0);
border-radius:4px;
border-right-color: rgba(0, 0, 0,40);
border-left-color: rgba(255, 255, 255,100);
border-top-color: rgba(255, 255, 153,0);
</string>
    </property>
    <property name="text">
     <string notr="true">5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="push9">
    <property name="geometry">
     <rect>
      <x>205</x>
      <y>245</y>
      <width>85</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bitstream Charter</family>
      <pointsize>22</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-image:url(/transparent.png);
background-color: rgba(139, 139, 139, 50);
border: 2px solid  rgb(0, 0, 0);
border-radius:4px;
border-right-color: rgba(0, 0, 0,40);
border-left-color: rgba(255, 255, 255,100);
border-top-color: rgba(255, 255, 153,0);
</string>
    </property>
    <property name="text">
     <string notr="true">9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="push7">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>245</y>
      <width>85</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bitstream Charter</family>
      <pointsize>22</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-image:url(/transparent.png);
background-color: rgba(139, 139, 139, 50);
border: 2px solid  rgb(0, 0, 0);
border-radius:4px;
border-right-color: rgba(0, 0, 0,40);
border-left-color: rgba(255, 255, 255,100);
border-top-color: rgba(255, 255, 153,0);
</string>
    </property>
    <property name="text">
     <string notr="true">7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="push8">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>245</y>
      <width>85</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bitstream Charter</family>
      <pointsize>22</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-image:url(/transparent.png);
background-color: rgba(139, 139, 139, 50);
border: 2px solid  rgb(0, 0, 0);
border-radius:4px;
border-right-color: rgba(0, 0, 0,40);
border-left-color: rgba(255, 255, 255,100);
border-top-color: rgba(255, 255, 153,0);
</string>
    </property>
    <property name="text">
     <string notr="true">8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushBack">
    <property name="geometry">
     <rect>
      <x>305</x>
      <y>50</y>
      <width>85</width>
      <height>56</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bitstream Charter</family>
      <pointsize>22</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-image:url(/transparent.png);
background-color: rgba(139, 139, 139, 50);
border: 2px solid  rgb(0, 0, 0);
border-radius:4px;
border-right-color: rgba(0, 0, 0,40);
border-left-color: rgba(255, 255, 255,100);
border-top-color: rgba(255, 255, 153,0);
</string>
    </property>
    <property name="text">
     <string notr="true">&lt;=</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushOk">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>375</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bitstream Charter</family>
      <pointsize>22</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-image:url(/transparent.png);
background-color: rgba(139, 139, 139, 50);
border: 2px solid  rgb(0, 0, 0);
border-radius:4px;
border-right-color: rgba(0, 0, 0,40);
border-left-color: rgba(255, 255, 255,100);
border-top-color: rgba(255, 255, 153,0);
</string>
    </property>
    <property name="text">
     <string notr="true">OK</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushCanc">
    <property name="geometry">
     <rect>
      <x>205</x>
      <y>375</y>
      <width>186</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bitstream Charter</family>
      <pointsize>22</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-image:url(/transparent.png);
background-color: rgba(139, 139, 139, 50);
border: 2px solid  rgb(0, 0, 0);
border-radius:4px;
border-right-color: rgba(0, 0, 0,40);
border-left-color: rgba(255, 255, 255,100);
border-top-color: rgba(255, 255, 153,0);
</string>
    </property>
    <property name="text">
     <string notr="true">CLR</string>
    </property>
   </widget>
   <widget class="QLabel" name="calcTitle">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>5</y>
      <width>471</width>
      <height>36</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bitstream Charter</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-image:url(/transparent.png);
background-color: rgba(255, 251, 123,40);
border: 2px solid  rgb(74, 74, 74);
border-radius:4px;
border-right-color: rgba(255, 255, 153,0);
border-left-color: rgba(255, 255, 153,0);
border-top-color: rgba(255, 255, 153,0);
</string>
    </property>
    <property name="text">
     <string notr="true">TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="push0">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>310</y>
      <width>85</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bitstream Charter</family>
      <pointsize>22</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-image:url(/transparent.png);
background-color: rgba(139, 139, 139, 50);
border: 2px solid  rgb(0, 0, 0);
border-radius:4px;
border-right-color: rgba(0, 0, 0,40);
border-left-color: rgba(255, 255, 255,100);
border-top-color: rgba(255, 255, 153,0);
</string>
    </property>
    <property name="text">
     <string notr="true">0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushDot">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>310</y>
      <width>85</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bitstream Charter</family>
      <pointsize>22</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-image:url(/transparent.png);
background-color: rgba(139, 139, 139, 50);
border: 2px solid  rgb(0, 0, 0);
border-radius:4px;
border-right-color: rgba(0, 0, 0,40);
border-left-color: rgba(255, 255, 255,100);
border-top-color: rgba(255, 255, 153,0);
</string>
    </property>
    <property name="text">
     <string notr="true">.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushSgn">
    <property name="geometry">
     <rect>
      <x>205</x>
      <y>310</y>
      <width>85</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bitstream Charter</family>
      <pointsize>22</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-image:url(/transparent.png);
background-color: rgba(139, 139, 139, 50);
border: 2px solid  rgb(0, 0, 0);
border-radius:4px;
border-right-color: rgba(0, 0, 0,40);
border-left-color: rgba(255, 255, 255,100);
border-top-color: rgba(255, 255, 153,0);
</string>
    </property>
    <property name="text">
     <string notr="true">+/-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushCls">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>50</y>
      <width>85</width>
      <height>56</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bitstream Charter</family>
      <pointsize>22</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-image:url(/transparent.png);
background-color: rgba(139, 139, 139, 50);
border: 2px solid  rgb(0, 0, 0);
border-radius:4px;
border-right-color: rgba(0, 0, 0,40);
border-left-color: rgba(255, 255, 255,100);
border-top-color: rgba(255, 255, 153,0);
</string>
    </property>
    <property name="text">
     <string notr="true">C</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushA">
    <property name="geometry">
     <rect>
      <x>305</x>
      <y>120</y>
      <width>85</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bitstream Charter</family>
      <pointsize>22</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-image:url(/transparent.png);
background-color: rgba(139, 139, 139, 50);
border: 2px solid  rgb(0, 0, 0);
border-radius:4px;
border-right-color: rgba(0, 0, 0,40);
border-left-color: rgba(255, 255, 255,100);
border-top-color: rgba(255, 255, 153,0);
</string>
    </property>
    <property name="text">
     <string notr="true">A</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushB">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>120</y>
      <width>85</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bitstream Charter</family>
      <pointsize>22</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-image:url(/transparent.png);
background-color: rgba(139, 139, 139, 50);
border: 2px solid  rgb(0, 0, 0);
border-radius:4px;
border-right-color: rgba(0, 0, 0,40);
border-left-color: rgba(255, 255, 255,100);
border-top-color: rgba(255, 255, 153,0);
</string>
    </property>
    <property name="text">
     <string notr="true">B</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushC">
    <property name="geometry">
     <rect>
      <x>305</x>
      <y>180</y>
      <width>85</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bitstream Charter</family>
      <pointsize>22</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-image:url(/transparent.png);
background-color: rgba(139, 139, 139, 50);
border: 2px solid  rgb(0, 0, 0);
border-radius:4px;
border-right-color: rgba(0, 0, 0,40);
border-left-color: rgba(255, 255, 255,100);
border-top-color: rgba(255, 255, 153,0);
</string>
    </property>
    <property name="text">
     <string notr="true">C</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushD">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>180</y>
      <width>85</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bitstream Charter</family>
      <pointsize>22</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-image:url(/transparent.png);
background-color: rgba(139, 139, 139, 50);
border: 2px solid  rgb(0, 0, 0);
border-radius:4px;
border-right-color: rgba(0, 0, 0,40);
border-left-color: rgba(255, 255, 255,100);
border-top-color: rgba(255, 255, 153,0);
</string>
    </property>
    <property name="text">
     <string notr="true">D</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushE">
    <property name="geometry">
     <rect>
      <x>305</x>
      <y>245</y>
      <width>85</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bitstream Charter</family>
      <pointsize>22</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-image:url(/transparent.png);
background-color: rgba(139, 139, 139, 50);
border: 2px solid  rgb(0, 0, 0);
border-radius:4px;
border-right-color: rgba(0, 0, 0,40);
border-left-color: rgba(255, 255, 255,100);
border-top-color: rgba(255, 255, 153,0);
</string>
    </property>
    <property name="text">
     <string notr="true">E</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushF">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>245</y>
      <width>85</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bitstream Charter</family>
      <pointsize>22</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-image:url(/transparent.png);
background-color: rgba(139, 139, 139, 50);
border: 2px solid  rgb(0, 0, 0);
border-radius:4px;
border-right-color: rgba(0, 0, 0,40);
border-left-color: rgba(255, 255, 255,100);
border-top-color: rgba(255, 255, 153,0);
</string>
    </property>
    <property name="text">
     <string notr="true">F</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushOX">
    <property name="geometry">
     <rect>
      <x>305</x>
      <y>310</y>
      <width>85</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bitstream Charter</family>
      <pointsize>22</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-image:url(/transparent.png);
background-color: rgba(139, 139, 139, 50);
border: 2px solid  rgb(0, 0, 0);
border-radius:4px;
border-right-color: rgba(0, 0, 0,40);
border-left-color: rgba(255, 255, 255,100);
border-top-color: rgba(255, 255, 153,0);
</string>
    </property>
    <property name="text">
     <string notr="true">0X</string>
    </property>
   </widget>
   <widget class="QLabel" name="risultatoText">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>50</y>
      <width>281</width>
      <height>56</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>UnDotum</family>
      <pointsize>28</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-image:url(/transparent.png);
background-color: qrgba(0, 0, 0, 100);
border: 2px solid  rgb(74, 74, 74);
border-radius:1px;
border-right-color: rgba(255, 255, 153,0);
border-left-color: rgba(255, 255, 153,0);
border-top-color: rgba(255, 255, 153,0);
</string>
    </property>
    <property name="text">
     <string notr="true"/>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
